at_secondary/at_secondary_server/lib/src/server/at_secondary_impl.dart

```diff
@@ -556,6 +559,23 @@ class AtSecondaryServerImpl implements AtSecondaryServer {
     await keyStore.deleteExpiredKeys();
   }
 
+  Future<void> _removeMalformedKeys() async {
+    try {
+      List<String> malformedKeys = AtSecondaryConfig.malformedKeysList;
+      logger.finest('malformed keys from config: $malformedKeys');
+      for (String key in malformedKeys) {
+        int? commitId = await _secondaryPersistenceStore
+            .getSecondaryKeyStore()!
+            .remove(key);
+        logger.finest('commitId for removed key $key: $commitId');
```

Comment:
(1) Only log if the key existed and was therefore deleted (2) log at WARNING if so

Reply:
done. please review